HW0/P5/P5.txt

```diff
@@ -1 +1,41 @@
 # Your answers here.
```

Comment:
Really great explanations in this problem. Helped me understand where I had originally gone wrong in my HW0.